js/main.js

```diff
@@ -82,6 +82,12 @@ async function main() {
           gridInput.checked = true;
           gridInput.dispatchEvent(new Event('change'));
         }
+        // Otherwise, default to pile and assign pile-display class by default
+        else {
+          const pileInput = document.getElementById("gacha-pile");
+          pileInput.checked = true;
+          pileInput.dispatchEvent(new Event('change'));
+        }
```

Comment:
does this need to be explicitly written? I assumed this would happen by default - what happens if we don't have this?

Reply:
Yeah. w/o this, the radio button would get selected visually but the change event that assigns the `pile-display` class to the appropriate container doesn't fire, which causes some weirdness when I tried to scale the page to portrait. I believe there are some specific CSS rules that looks for `.pile-display`.